app/src/main/java/com/locoquest/app/dao/IBenchmarkDAO.kt

```diff
@@ -14,29 +14,39 @@ interface IBenchmarkDAO {
     fun getBenchmarkByPid(@Query("pid") pid: String): Call<Benchmark>
 }
 
-fun main() {
-    val retrofit = Retrofit.Builder()
-        .baseUrl("https://geodesy.noaa.gov/api/nde/")
+fun createRetrofitInstance(baseUrl: String): Retrofit {
+    return Retrofit.Builder()
+        .baseUrl(baseUrl)
         .addConverterFactory(MoshiConverterFactory.create())
         .build()
-
+}
```

Comment:
Retrofit builder can be turned into a function, incase multiple API's are needed.
https://dev.to/levivm/coding-best-practices-chapter-one-functions-4n15

Reply:
I'd keep this as it were.  The Retrofit factory belongs in RetrofitClientInstance.